components/courses/spec/features/admin/progress/update_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'Progress UPDATE' do
+  describe 'validations' do
+    it { is_expected.to allow_values(::Courses::Progress::ALLOWED_MARKS).for(:homework_mark) }
+    it { is_expected.to_not allow_value('20').for(:homework_mark) }
+    it { is_expected.to_not allow_value('10').for(:homework_mark) }
+    it { is_expected.to_not allow_value('5').for(:homework_mark) }
+    it { is_expected.to_not allow_value('-2').for(:homework_mark) }
```

Comment:
All these values (-2, 5, 10, 20) - could also be wrapped in an array, instead of each being a separate line?

```
it { is_expected.to_not allow_values(%w(-2 5 10 20)).for(:homework_mark) }
```

Reply:
It is outdated file